package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "bookbinder",
-  "version": "1.3.7",
+  "version": "1.5.0",
```

Comment:
😂 😭  I just got confused for 5 min there because I thought this was `package.json` -- this is so not my area 🙈